rollkit/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## Unreleased
 
+## [`v0.3.0`](https://github.com/ignite/apps/releases/tag/rollkit/v0.3.0)
```

Comment:
I will tag this only when github.com/rollkit/rollkit and github.com/rollkit/go-execution-abci have tags